packs/data/actions.db/mark-for-death.json

```diff
@@ -16,7 +16,146 @@
         "requirements": {
             "value": ""
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.MarkForDeath.TargetMark",
+                "option": "marked-for-death",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.MarkForDeath.TargetMark",
+                "predicate": {
+                    "all": [
+                        "action:seek",
+                        "marked-for-death"
+                    ]
+                },
+                "selector": "perception",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.SpecificRule.MarkForDeath.TargetMark",
+                "predicate": {
+                    "all": [
+                        "action:feint",
+                        "marked-for-death"
+                    ]
+                },
+                "selector": "deception",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "definition": {
+                    "any": [
```

Comment:
I think this ability should only work on unarmed attacks that are also agile or finesse. As worded, it will work on any unarmed attack.

Reply:
Removed the unarmed predicate in the new commit.